client/driver/lxc_test.go

```diff
@@ -38,23 +39,39 @@ func TestLxcDriver_Fingerprint(t *testing.T) {
 	node := &structs.Node{
 		Attributes: map[string]string{},
 	}
-	apply, err := d.Fingerprint(&config.Config{}, node)
-	if err != nil {
-		t.Fatalf("err: %v", err)
-	}
-	if !apply {
-		t.Fatalf("should apply by default")
-	}
 
-	apply, err = d.Fingerprint(&config.Config{Options: map[string]string{lxcConfigOption: "0"}}, node)
-	if err != nil {
-		t.Fatalf("err: %v", err)
-	}
-	if apply {
-		t.Fatalf("should not apply with config")
+	// test with an empty config
```

Comment:
consider turning these into subtests with a test case struct to capture inputs and expected output